src/style.css

```diff
@@ -0,0 +1,132 @@
+@font-face {
+  font-family: Roboto;
+  src: url(Roboto/Roboto-Medium.ttf);
+  font-style: normal;
+}
+
+.body {
+  margin: 0;
+}
+
+.card {
+  width: 198px;
+  height: 406px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  box-sizing: border-box;
+}
+
+.img {
+  margin-top: 32px;
+}
+
+.logo {
+  width: 160px;
+  height: 134px;
+}
+
+.card__name {
+  margin-top: 40px;
+
+  width: 166px;
+  height: 36px;
```

Comment:
do not hardcode size for content - it should have 100% width
also you may use paddings for card to add indentation from border